src/main/java/com/adobe/target/edge/client/service/DefaultTargetService.java

```diff
@@ -257,4 +263,20 @@ private CompletableFuture<ResponseWrapper<DeliveryResponse>> callDeliveryApiAsyn
         deliveryRequest.getDeliveryRequest(),
         DeliveryResponse.class);
   }
+
+  private void cleanUpGeoContext(TargetDeliveryRequest deliveryRequest) {
+    Geo geo = deliveryRequest.getDeliveryRequest().getContext().getGeo();
```

Comment:
Is there any possibility that getContext() could return null here, resulting in NPE?